examples/Identifying Local Environments in a Complex Crystal.ipynb

```diff
@@ -24,7 +24,9 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    neighbor = {"r_max": first_well}
</pre><p>This variable now has the same name as the one before it which changes what the call to compute is doing. If you change the name of the variable, that should fix it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>